src/modules/game/MedalOfHonorTracker.js

```diff
@@ -11,23 +11,24 @@ class MedalOfHonorTracker extends HFX.Feature {
         profile: "https://hackforums.net/member.php?action=profile&uid=585389"
       }
     });
+    this.numberFormatter = new Intl.NumberFormat("en-US");
```

Comment:
I have never heard of or seen this library before. But holy damn that is useful.